src/server_manager/ui_components/outline-server-view.html

```diff
@@ -632,6 +631,17 @@ <h3>[[localize('data-limits-dialog-title')]]</h3>
         // Push to array using Polymer in order to propagate changes. See
         // https://www.polymer-project.org/1.0/docs/devguide/model-data#array-mutation
         this.push('accessKeyRows', accessKey);
+        // Force rendering to happen synchronously.  Otherwise we will finish the loop before the new DOM element is added
+        this.$.accessKeysContainer.querySelector('dom-repeat').render();
```

Comment:
This is a bit fragile, as it depends on the HTML structure. Checking the placeholder is also unfortunate, since you actually have an id. The placeholder could simply be "new key" which would break us.

I believe you can go over the elements in the dom-repeat, check the id of the corresponding item, and set the focus on the input. There are a number of methods, like modelForElement, get, itemForElement.... Let's look into those.

Reply:
Some references:
https://polymer-library.polymer-project.org/2.0/docs/devguide/templates
https://polymer-library.polymer-project.org/2.0/api/elements/Polymer.DomRepeat

Reply:
Thinking more about this.
In the HTML template, you could set an id to each entry. Something like `key-{id}`. Then you could search by id.

Otherwise, an alternative to relying on the placeholder name is to call itemForElement(el).id